create_marc_record.py

```diff
@@ -0,0 +1,136 @@
+from pymarc import Record, Field, Subfield
+
+
+def create_base_record() -> Record:
+    """Create a base MARC record, to which metadata from an external
+    source will be added."""
+    record = Record()
+
+    # Leader fixed field.
+    # Type (Leader/06) - j (Musical sound recording)
+    record.leader.type_of_record = "j"
+    # BLvl (Leader/07) - m (Monographic/Item)
+    record.leader.bibliographic_level = "m"
+    # Ctrl (Leader/08) - # (No specified type)
+    record.leader.type_of_control = "#"
```

Comment:
This value needs to be a blank (an actual `" "`, ASCII 32).  When you see `#` in MARC documentation, it means a blank, just as you did with the indicators for the data fields.  See https://www.loc.gov/marc/specifications/specrecstruc.html#define for more.

Pymarc accepts a literal `#`, but does not convert it to a space:
```
>>> sample.leader.leader
'      jm#a22     3i 4500'
```